pdsfile/pds3file/rules/JNOSRU_xxxx.py

```
@@ -0,0 +1,105 @@
+##########################################################################################
```

Ruff (0.14.7)

The file is executable but no shebang is present

(EXE002)